src/Augurk.UI/Components/FeatureResult.razor

```diff
@@ -10,10 +10,11 @@
  Unless required by applicable law or agreed to in writing, software
```

Comment:
Update license to include 2021

Reply:
As discussed we've simplified the license header to not include a year